CI/check-services.py

```diff
@@ -95,7 +102,9 @@ def check_rtmp_server(uri) -> bool:
     else:
         port = 1935
 
+    ip_address = '[none]'
     try:
+        ip_address = socket.gethostbyname(hostname)
```

Comment:
Technically this IP could end up being a different one than what `create_connection()` resolves to, to ensure they're the same replace `hostname` with `ip_address` when the connection is created below.